modules/dists/StencilDist.chpl

```diff
@@ -1531,6 +1464,7 @@ iter StencilArr.dsiBoundaries(param tag : iterKind) where tag == iterKind.standa
 //
 // Copies the range slice function in ChapelArray.
 //
+pragma "no copy return"
```

Comment:
Long-term, yes. For this effort I attempted to do so but observed an increase in leaks. I'd like attempt more, but would rather leave that as a post-merge item.